stock_by_warehouse_sale/models/sale_order_line.py

```diff
@@ -4,7 +4,11 @@
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    warehouses_stock = fields.Text(store=False, readonly=True)
+    warehouses_stock = fields.Text(
+        store=False,
+        readonly=True,
+        compute="_compute_warehouse_stock",
```

Comment:
Same here, consider JSON type and store/readonly are not required.

Reply:
Done